.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 test/fixtures/*.actual.css
+.idea/
```

Comment:
Well, it is a matter of taste I think. I do not expect everyone to have a proper global gitignore. But I can remove the commit if this is the only problem with the pull request.